core/src/main/scripts/importer/validateData.py

```diff
@@ -2094,7 +2094,7 @@ def parseFeatureColumns(self, nonsample_col_vals):
         # products if the antibody name has a particular format.
         value = nonsample_col_vals[0].strip()
         if '|' not in value:
-            self.logger.error('No pipe symbol in RPPA probe column',
+            self.logger.error('No pipe symbol in Composite.Element.REF probe column',
```

Comment:
I'll add it to the list of things to change in an updated version of the CPTAC data.